tests/components/geonetnz_volcano/test_config_flow.py

```diff
@@ -23,7 +23,7 @@ async def test_duplicate_error(hass, config_entry):
     flow.hass = hass
 
     result = await flow.async_step_user(user_input=conf)
-    assert result["errors"] == {"base": "identifier_exists"}
+    assert result["errors"] == {"base": "already_configured_location"}
```

Comment:
This should use `already_configured` key and a reference to `already_configured_location` common key.

Please make a separate PR for this change.

Reply:
ok

Reply:
I'll do it here https://github.com/home-assistant/core/pull/41981